### DISCUSSION TITLE
New Advertising Problem

### DISCUSSION THREAD

**Discussion by LWFlouisa** [FreeTubeApp/FreeTube#5640] [General]
What would be the solution for channels that don't use Youtube ads, but specifically have adverts in the video itself? This seems to be becoming more and more of a problem.

For example this one Art History channels has ads in the video itself.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
You can enable SponsorBlock under Settings -> SponsorBlock Settings -> `Enable SponsorBlock`
